vcs/vcs2vtk.py

```diff
@@ -29,7 +29,7 @@
     "eqdc",
     "tmerc",
     "stere",
-    "lcca",
```

Comment:
was it always wrong? Or has it been updated? Should we keep `lcca` as valid but swap it to `laea` under the hood while issuing a warning?